2-multitenant/envs/non-production/README.md

```diff
@@ -0,0 +1,20 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| envs | Environments | <pre>map(object({<br>    billing_account    = string<br>    folder_id          = string<br>    network_project_id = string<br>    network_self_link  = string<br>    org_id             = string<br>    subnets_self_links = list(string)<br>  }))</pre> | n/a | yes |
+
+## Outputs
+
+| Name | Description |
```

Comment:
I think it's missing an output for the DB. Otherwise how do we refer to it later?

Reply:
I suspect the name is hardcoded into the K8s manifests rather than variable? @yliaog?

That said if we keep the DB creation in this stage I agree we should included in the output.

Reply:
Added DB self_links output

Reply:
The DBs will be moved to phase 4 in a different PR.